models.py

```diff
@@ -38,9 +43,8 @@ def arima(X):
     with warnings.catch_warnings():
         warnings.filterwarnings("ignore")
         for k, g in grouped:
-            g = g[-5:]
             gi = g.interpolate(method = 'linear', limit = 50, limit_direction = 'backward')
-            model = ARIMA(gi, order=(2,1,0))
+            model = ARIMA(gi.as_matrix(), order=(1,2,0))
```

Comment:
Changes to the ARIMA model should be moved to another PR, since this PR is just for the status quo model.